### ISSUE TITLE
Feature to add poddistributionbudget as part of the helm chart

### ISSUE THREAD

**Issue by prasanna-ranga** [neo4j/helm-charts#249]
**Is your feature request related to a problem? Please describe.**
A template for the PodDistributionBudget would be good 

**Describe the solution you'd like**
```
{{- if .Values.podDisruptionBudget.enabled }}
---
apiVersion: {{ .Capabilities.APIVersions.Has "policy/v1" | ternary "policy/v1" "policy/v1beta1" }}
kind: PodDisruptionBudget
metadata:
  name: {{ include "neo4j.fullname" . }}
  namespace: {{ .Values.namespace }}
  labels:
    app: "{{ template "neo4j.name" $ }}"
    {{- include "neo4j.labels" $.Values.neo4j | indent 4 }}
spec:
  {{- if .Values.podDisruptionBudget.minAvailable }}
  minAvailable: {{ .Values.podDisruptionBudget.minAvailable }}
  {{- end }}
  {{- if .Values.podDisruptionBudget.maxUnavailable }}
  maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
  {{- end }}
  selector:
    matchLabels:
        app: "{{ template "neo4j.name" . }}"
        helm.neo4j.com/instance: "{{ include "neo4j.fullname" . }}"
{{- end }}
```

**Describe alternatives you've considered**

**Additional context**

**Comment by harshitsinghvi22** [comment]
Implemented in v5 via below PR
v5 - https://github.com/neo4j/helm-charts/pull/254

**Comment by harshitsinghvi22** [comment]
Implemented in v4.4 via below PR
https://github.com/neo4j/helm-charts/pull/255
